src/electrai/entrypoints/train.py

```diff
@@ -11,10 +11,20 @@
 from src.electrai.dataloader.registry import get_data
 from src.electrai.lightning import LightningGenerator
 from torch.utils.data import DataLoader
+from torch.utils.data._utils.collate import default_collate
 
 torch.backends.cudnn.conv.fp32_precision = "tf32"
 
 
+def collate_fn(batch):
+    try:
+        return default_collate(batch)
+    except Exception:
```

Comment:
This catches all possible exceptions, which is very broad. Some of them could be real errors that should be thrown. Is there a particular class of exceptions that you want to catch here?